doc/specs/vulkan/chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -115,13 +115,14 @@ include::../../api/structs/VkSwapchainCreateInfoKHR.txt[]
     The swapchain is associated with pname:surface.
   * pname:minImageCount is the minimum number of presentable images that the
     application needs.
-    The platform will either create the swapchain with at least that many
-    images, or will fail to create the swapchain.
-  * pname:imageFormat is a slink:VkFormat that is valid for swapchains on
-    the specified surface.
-  * pname:imageColorSpace is a slink:VkColorSpaceKHR that is valid for
-    swapchains on the specified surface.
-  * pname:imageExtent is the non-zero size (in pixels) of the swapchain.
+    The implementation will either create the swapchain with at least that
+    many images, or it will fail to create the swapchain.
+  * pname:imageFormat is a slink:VkFormat that the swapchain image(s) will
+    be created with.
+  * pname:imageColorSpace is a slink:VkColorSpaceKHR that identifies how
```

Comment:
Typo "how will"

Reply:
Probably will change whole for better estheticity as @TobiasHector suggests below.

But out of curiosity, what is wrong with "how will"?

Reply:
... identifies *how* (the swapchain) *will* interpret. I don't know the right grammar term for this but the current usage isn't good.

Reply:
I agree with @oddhack.  I like changing it to "... identifies how the swapchain will ..."